crates/hyperqueue/src/common/cli.rs

```diff
@@ -199,7 +199,7 @@ pub enum SubCommand {
     /// Commands for controlling HyperQueue workers
     Worker(WorkerOpts),
     /// Operations with log
-    Log(LogOpts),
```

Comment:
Should this be called `Stream`? `Read` seems quite general, we might want to introduce more things that could be read by the server in the future.

Reply:
I did not want name it just "stream" as it is a verb for creating a stream. But "read-stream" seems quite long for me so I have left just "read".

Reply:
Our existing commands use `<noun> <verb>`:
- `server start` (noun verb)
- `job submit` (noun verb)
There are some cases that are something between a noun and a verb (summary, info, list), but it's mostly consistent.

So to keep it consistent, this should be `hq stream read`, IMO.

Reply:
I do not like it as we are not planning any other `hq stream <something>`. If you think that noun is better than I propose `hq stream-reader` .

Reply:
That sounds surprising, as we have several operations with the log. I checked the CLI help now and I think that we have unnecessarily moved one layer down.

Now there is `hq read cat` , `hq read export`, `hq read jobs`, that is really weird. I understand that you don't want `hq stream read cat`, but if we just get rid of `read`, I think that it will solve itself. So we should have just `hq stream export`, `hq stream cat` etc.

Reply:
Sorry for briefness. I wanted to propose `hq stream-reader cat ...`.  I just want to avoid name "stream" as it seems to me very misleading, I want to signal that is just a reading end of the interface, not "streaming" one.

Reply:
Renamed to `hq outout-log`